src/overlays/actors/ovl_Obj_Warp2block/z_obj_warp2block.c

```diff
@@ -43,36 +43,286 @@ const ActorInit Obj_Warp2block_InitVars = {
     (ActorFunc)ObjWarp2block_Update,
     (ActorFunc)ObjWarp2block_Draw,
 };
-*/
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Warp2block/func_80BA1DF0.s")
+typedef struct {
+  f32 scale;
+  f32 focus;
+  s16 params;
+} Warp2BlockSpawnData;
+
+ Warp2BlockSpawnData sSpawnData[2] = {
+    { 1.0f, 60.0f, 0x0018 },
+    { 0.6f, 40.0f, 0x0019 },
+};
+
+f32 sDistances[] = {60.0f, 100.0f, 140.0f, 180.0f, 220.0f, 260.0f, 300.0f, 300.0f};
+
+static InitChainEntry sInitChain[] = {
+    ICHAIN_F32(uncullZoneForward, 1800, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 300, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneDownward, 1500, ICHAIN_STOP),
+};
+
+Color_RGB8 sColors[] = {
+    {100, 120, 140}, { 80, 140, 200}, {100, 150, 200}, {100, 200, 240},
+    { 80, 110, 140}, { 70, 160, 225}, { 80, 100, 130}, {100, 110, 190},
```

Comment:
make sure to run the formatter.
if you havent already youll need to install `clang-format` and `clang-tidy` via apt-get

Reply:
Weird, I did run the formatter (running it again now doesn't change the file at all). Is it still `./format.sh`? Otherwise, anything I should change here?

Reply:
it is. are you sure you have the two programs mentioned above installed? 
there are various formatting issues like pointer stars on the right, braces without proper spacing etc. would be easier to figure out why the formatter isnt working than to try to fix every little thing

Reply:
just to be sure, make sure you run 
`sudo apt-get install clang-format`
`sudo apt-get install clang-tidy`

Reply:
Damn, my bad. Turns I didn't have them installed, but the format script's output still says "running clang-tidy..." or something similar so I was assuming it was running properly.

Anyways, running it on the whole repo changes a *bunch* of files I'm not otherwise touching in this PR. Not sure if that's expected, but I just ran it on the files I'm changing to avoid filling up this

Reply:
running it on the whole repo is fine, though i just ran it myself on recently merged prs. I think this is related to the strange oddity of the formatter being inconsistent depending on who runs it lol (@Roman971 we gotta figure this out some time)